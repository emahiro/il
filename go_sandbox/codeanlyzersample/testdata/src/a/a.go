package a

import "context"

func main() {
	Sample()
	SampleWithContext(context.Background())
}

func Sample() error { // want `change Sample to Example`
	return nil
}

func SampleWithContext(ctx context.Context) error { // want `change SampleWithContext to ExampleWithContext`
	return nil
}
