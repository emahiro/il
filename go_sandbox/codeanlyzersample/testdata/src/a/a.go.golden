package a

import "context"

func main() {
	Sample()
	SampleWithContext(context.Background())
}

func Sample() error {
	return nil
}

func SampleWithContext(ctx context.Context) error {
	return nil
}
